lib/Reporting/ReportType.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Stripe\Reporting;
+
+/**
+ * Class ReportType
+ *
+ * @property string $id
+ * @property string $object
+ * @property int $data_available_end
+ * @property int $data_available_start
+ * @property string $name
+ * @property int $updated
+ * @property string $version
+ * @property mixed $result
+ * @property string $status
+ * @property int $succeeded_at
```

Comment:
I'm not seeing `result`/`status`/`succeeded_at` in the implementation. Are you sure those are still relevant?

Reply:
hmmm looks like a bad copy paste, fixing

Reply:
Fixed